make_dataset.py

```diff
@@ -305,10 +297,8 @@ def parse_args(args_input=None):
     # Warn user they specified kwargs for degradation not being used
     for deg, args in degradation_kwargs.items():
         if deg not in ARGS.degradations and len(args) > 0:
-            warnings.warn(
-                "--degradation_kwargs contains args for unused "
-                f'degradation "{deg}".',
-                UserWarning,
+            logging.warning(
+                "--degradation_kwargs contains args for unused " f'degradation "{deg}".'
```

Comment:
Can just be one f-string as opposed to two adjacent

Reply:
Done. Also fixed where this happened elsewhere due to the auto-formatter.